src/hor2eq.jl

```diff
@@ -85,27 +89,29 @@ It performs precession, nutation, aberration, and refraction corrections.
   `true` by default
 * `aberration` (optional boolean keyword): set this to `false` for no aberration
   correction, `true` by default
-* `refract` (optional boolean keyword: set this to `false` for no refraction
+* `refract` (optional boolean keyword): set this to `false` for no refraction
   correction, `true` by default
 * `lat` (optional keyword): north geodetic latitude of location, in degrees. Default
-  is NaN
+  is `NaN`
 * `lon` (optional keyword): AST longitude of location, in degrees. You can specify west
-  longitude with a negative sign. Dafault is NaN.
+  longitude with a negative sign. Default value is `NaN`
 * `altitude` (optional keyword): the altitude of the observing location, in meters.
-  It it zero by default
+  It is `0` by default
 * `pressure` (optional keyword): the pressure at the observing location, in millibars.
-  Default is NaN
+  Default value is `NaN`
 * `temperature` (optional keyword): the temperature at the observing location, in Kelvins.
-  Default is NaN
+  Default value is `NaN`
 * `obsname` (optional keyword): set this to a valid observatory name to
-  be used by the observatory type in [types](@ref), which will return
-  the latitude and longitude to be used by this program. Default is NaN.
+  be used by the observatory type in [types](@ref), which will return the latitude and
+  longitude to be used by this program. This is `""` (empty string) by default,
+  in which case `lat` and `lon` default to the coordinates of the `Pine Bluff Observatory`
+  provided they are equivalent to `NaN` individually
 
 ### Output ###
 
 * `ra`: right ascension of object, in degrees (FK5)
 * `dec`: declination of the object, in degrees (FK5)
-* `ha`: hour angle, in degrees
+* `ha`: hour angle, in hours
```

Comment:
Why this change?  I'd have kept the unit in degrees, for consistency with IDL AstroLib